link-checker.js

```diff
@@ -75,14 +85,36 @@ module.exports = function(directory, options = {}, callback) {
 		fileCounter += 1
 		localPages.add(filePath)
 
-		const $ = cheerio.load(fileContent)
+
+		const lines = fileContent.split('\n');
+		const lineStarts = [];
+		for(let i=0, e=lines.length; i<e; i++) {
```

Comment:
No need to assign the length to `e` here.

Reply:
```suggestion
		for(let i=0; i<lines.length; i++) {
```